syntax = "proto3";

package tesla.proto.energy_device.v1;

option go_package = "github.com/teslamotors/energy_device/pkg/protocol/protobuf/energydevice/v1";
option java_package = "com.tesla.generated.energy_device.v1";
option java_outer_classname = "AuthorizationRole";

enum AuthorizationRole {
  AUTHORIZATION_ROLE_INVALID  = 0;
  AUTHORIZATION_ROLE_CUSTOMER = 1;
  AUTHORIZATION_ROLE_VEHICLE  = 2;
}